### PR TITLE
[RA1 Ch01] Abbreviations table added

### PR DESCRIPTION
**Pull request by karinesevilla**
Fixes issue #3107

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 1]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+   :widths: 20 60
+   :header-rows: 1
+
+   * - Abbreviation/Acronym
+     - Definition
+   * - API
+     - Application Programming Interface
+   * - DNS
+     - Domain Name System
+   * -  DPDK
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:167]:
```suggestion
   * - DPDK
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 2]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+     - Data Plane Development Kit
+   * - ECMP
+     - Equal Cost Multi     - Path routing
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:170]:
```suggestion
     - Equal Cost Multi-Path routing
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 3]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+   * - ETSI
+     - European Telecommunications Standards Institute
+   * - FPGA
+     - Field Programmable Gate Array
+   * - GB/TB
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:175]: are we using GiB or MiB or TiB anywhere?

Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:175]:
```suggestion
   * - MB/GB/TB
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 4]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+     - GigaByte/TeraByte
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:176]:
```suggestion
     - MegaByte/GigaByte/TeraByte
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 5]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+
+.. list-table:: 
+   :widths: 20 60
+   :header-rows: 1
+
+   * - Abbreviation/Acronym
+     - Definition
+   * - API
+     - Application Programming Interface
+   * - DNS
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:165]:
```suggestion
   * - CPU
     - Central Processing Unit
   * - DNS
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 6]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+     - European Telecommunications Standards Institute
+   * - FPGA
+     - Field Programmable Gate Array
+   * - GB/TB
+     - GigaByte/TeraByte
+   * - GPU
+     - Graphics Processing Unit
+   * - GRE
+     - Generic Routing Encapsulation
+   * - GSMA
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:181]:
```suggestion
   * - GSM
     - Global System for Mobile Communications (originally Groupe Spécial Mobile)
   * - GSMA
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 7]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+   * - ICMP
+     - Internet Control Message Protocol
+   * - IO
+     - Input/Output
+   * - IOPS
+     - Input/Output per Second
+   * - IPMI
+     - Intelligent Platform Management Interface
+   * - KVM
+     - Kernel - based Virtual Machine
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:208]:
```suggestion
     - Kernel-based Virtual Machine
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 8]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+   * - LFN
+     - Linux Foundation Networking
+   * - LMA
+     - Logging, Monitoring and Analytics
+   * - LVM
+     - Logical Volume Management
+   * - MANO
+     - Management ANd Orchestration
+   * - MLAG
+     - Multi - chassis Link Aggregation Group
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:222]:
```suggestion
     - Multi-chassis Link Aggregation Group
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 9]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+   * - NFVI
+     - Network Function Virtualisation Infrastructure
+   * - NIC
+     - Network Interface Card
+   * - NPU
+     - Numeric Processing Unit
+   * - NTP
+     - Network Time Protocol
+   * - NUMA
+     - Non - Uniform Memory Access
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:238]:
```suggestion
     - Non-Uniform Memory Access
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 10]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+     - Operating System
+   * - OSTK
+     - OpenStack
+   * - OVS
+     - Open vSwitch
+   * - OWASP
+     - Open Web Application Security Project
+   * - PCIe
+     - Peripheral Component Interconnect Express
+   * - PCI - PT
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:249]:
```suggestion
   * - PCI-PT
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 11]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+     - PCIe PassThrough
+   * - PXE
+     - Preboot Execution Environment
+   * - QoS
+     - Quality of Service
+   * - RA
+     - Reference Architecture
+   * - RA - 1
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:257]:
```suggestion
   * - RA-1
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 12]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+     - Reference Architecture - 1
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:258]:
```suggestion
     - Reference Architecture 1 (i.e., Reference Architecture for OpenStack-based Cloud Infrastructure)
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 13]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+   * - RBAC
+     - Role - based Access Control
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:260]:
```suggestion
     - Role-based Access Control
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 14]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+   * - SDN
+     - Software Defined Networking
+   * - SFC
+     - Service Function Chaining
+   * - SLA
+     - Service Level Agreement
+   * - SMP
+     - Symmetric Multiprocessing
+   * - SMT
+     - Simultaneous multithreading
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:280]:
```suggestion
     - Simultaneous MultiThreading
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 15]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+   * - SAST
+     - Static Application Security Testing
+   * - SDN
+     - Software Defined Networking
+   * - SFC
+     - Service Function Chaining
+   * - SLA
+     - Service Level Agreement
+   * - SMP
+     - Symmetric Multiprocessing
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:278]:
```suggestion
     - Symmetric MultiProcessing
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 16]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+   * - SMT
+     - Simultaneous multithreading
+   * - SNAT
+     - Source Network Address Translation
+   * - SNMP
+     - Simple Network Management Protocol
+   * - SR - IOV
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:285]:
```suggestion
   * - SR-IOV
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 17]
```diff
@@ -151,6 +151,170 @@ General terminology definitions can be found in
...
+     - Logging, Monitoring and Analytics
+   * - LVM
+     - Logical Volume Management
+   * - MANO
+     - Management ANd Orchestration
+   * - MLAG
+     - Multi - chassis Link Aggregation Group
+   * - NAT
+     - Network Address Translation
+   * - NFS
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:225]: Chapter 2 uses NF -- I think it should be VNF but if not then
```suggestion
   * - NF
     - Network Function
   * - NFS
```

**Review by pgoyal01 — CHANGES_REQUESTED** [review]

**Review by pgoyal01 — APPROVED** [review]

**Review by ildikov — APPROVED** [review]

**Review by collivier — APPROVED** [review]

**Pull request merged**